src/Test/L0/Listener/Configuration/Testable/TestWindowsServiceControlManager.cs

```diff
@@ -0,0 +1,28 @@
+using System.ServiceProcess;
+
+using Microsoft.VisualStudio.Services.Agent.Listener.Configuration;
+
+namespace Test.L0.Listener.Configuration.Testable
+{
+    public class TestWindowsServiceControlManager : WindowsServiceControlManager
+    {
+        public ServiceController SelectedServiceController { get; set; }
+
+        protected override void InstallService(
```

Comment:
IWindowsSecurityManager is essentially is that one, but I kept it all security related win32 apis in that.. the few left over apis are stuck in windowsServiceControlManager as virtual methods. I guess I will move all these virtual methods to IWindowSecutiryManager and rename that to INativeWindowsServiceHelper. (INativeLinuxServiceHelper for linux)